docs/policy_templates_data/policy_templates.json

```diff
@@ -1239,6 +1239,34 @@
           }
         ]
       }
+    },
+    "FilterLogEventsPolicy": {
+      "Description": "Gives permission to filter Log Events from any Log Group",
```

Comment:
Update the description since you change the Resource to scope down to one log group.

Reply:
Good catch. Thanks